lib/index.js

```diff
@@ -156,7 +159,15 @@ UI.prototype.prependLine = function(prependData, data, prepend) {
   @param {Error} error
 */
 UI.prototype.writeError = function(error) {
-  writeError(this, error);
+  const report = writeError(this, error);
+  if (report === null) {
+    return;
+  }
+  const md5 = crypto.createHash('md5').update(report).digest("hex");
+  const filepath = `${tmpdir}/error.dump.${md5}.log`;
```

Comment:
not 100% sure but will "/" work on Windows? or maybe we should use something like `path.join(tmpdir, 'error.dump.${md5}.log')`?

Reply:
all `fs.*` methods automagically transform `/` to the appropriate slash. Our code can be all written posix style.